### PR TITLE
Screener doctor model

### PR DESCRIPTION
**Pull request by bucephalus**
for: https://github.com/CS169-Raxa/raxa-visualizations/issues/127

anything I should add/change?

### CODE REVIEW

File: app/models/doctor.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Doctor < ActiveRecord::Base
+  attr_accessible :max_workload, :name, :specialty
```
Comment (danwang) [inline app/models/doctor.rb]: For specialty, should we use a model for specialty, like we did for department?

**Comment by danwang** [comment]
Otherwise, everything looks good.

File: app/models/doctor.rb (hunk 1 above)
Comment (bucephalus) [inline app/models/doctor.rb]: Hmm.  It is the same issue, since we'll need all the specialties to make the dropdown menu.  That sounds like a good idea.

Comment (bucephalus) [inline app/models/doctor.rb]: Also just realized that doctors may have multiple specialties

Comment (danwang) [inline app/models/doctor.rb]: Please add this model to this pull request.

Comment (bucephalus) [inline app/models/doctor.rb]: was working on it.  added now

**Comment by danwang** [comment]
LOOKS GOOD TO ME

**Pull request merged**